### DISCUSSION TITLE
dom_designer/14.0.0/basic/H_ABOUT_RUNNING_RESTRICTED_AGENTS

### DISCUSSION THREAD

**Discussion by giscus** [HCL-TECH-SOFTWARE/HCLSoftware-Documentation-Home#97] [Q&A]
# dom_designer/14.0.0/basic/H_ABOUT_RUNNING_RESTRICTED_AGENTS

Note that restricted operations are available only to users with unrestricted access; restricted operations are not available to users with restricted access.

https://help.hcl-software.com/dom_designer/14.0.0/basic/H_ABOUT_RUNNING_RESTRICTED_AGENTS.html

**Comment by mariobareither** [comment]
Seems that "session.CreateHTTPRequest" is missing in this list (since v10.0.1); apparently, this call performs a disk i/o operation.
Please cross-check and update if applicable.
